src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java

```diff
@@ -0,0 +1,44 @@
+package com.google.checkstyle.test.chapter4formatting.rule4854fieldannotations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class InputFieldAnnotations {
+    public @interface SomeAnnotation1 { }
```

Comment:
please add few methods in this Input to make it clear that behavior on fields is different,

Reply:
done

https://github.com/checkstyle/checkstyle/blob/43e4d77faad75a1e80686f55b5e35c1622cc541a/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java#L45-L53